src/test/java/itest/util/Utils.java

```diff
@@ -50,7 +59,63 @@ public class Utils {
 
     private static final Vertx VERTX =
             Vertx.vertx(new VertxOptions().setPreferNativeTransport(true));
-    public static final HttpClient HTTP_CLIENT = VERTX.createHttpClient(HTTP_CLIENT_OPTIONS);
+    public static final HttpClient HTTP_CLIENT =
+            VERTX.createHttpClient(HTTP_CLIENT_OPTIONS)
+                    .redirectHandler(
+                            resp -> {
```

Comment:
As we talked, it seems that it can not send the body to the redirected url. Do you want to keep it in the code?